### ISSUE TITLE
Error syincing design via photoshop extension 

### ISSUE THREAD

**Issue by waituk** [avocode/avocode#400]
It happened when simply clicking on the sync tab on photoshop plugin. It got stuck at 100%  and then showed the error occured warning.

-----
App v.2.3.2-ea60620
Source UID: 42776
```
Error: incorrect header check
    at Zlib._handle.onerror (zlib.js:345:17)
```

**Comment by rychlis** [comment]
sorry for the late response. Did this error really happen in Photoshop? It seems like an error report in the Avocode App. Can you reproduce it in the latest avocode 2.4.3?

**Comment by rychlis** [comment]
I am closing this issue as it's a duplicate of #447. Thanks for your report.

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.
